server/config/routes.js

```diff
@@ -93,6 +98,41 @@ exports.configure = function (app) {
   app.get('/location/province/:uuid', locations.lookupProvince);
   app.get('/location/detail/:uuid', locations.lookupDetail);
 
+  //API for account routes crud
+  app.get('/accounts/:list_type', account.list);
+  app.get('/account/:id', account.getAccount);
+  app.post('/accounts', account.create);
+  app.put('/accounts/:id', account.update);
+
+  //API for account type routes crud
+  app.get('/account_types', accountType.list);
+  app.get('/account_types/:id', accountType.getAccountType);
+  app.post('/account_types', accountType.create);
+  app.put('/account_types/:id', accountType.update);
+  app.delete('/account_types/:id', accountType.remove);
+
+  //API for cost_center routes crud
+  app.get('/cost_centers/:list_type', costCenter.list);
+  app.get('/cost_center/:id', costCenter.getCostCenter);
```

Comment:
This should use the same root URL as the other routes in the `costCenter` controller.